.envrc

```
@@ -188,12 +188,20 @@ make setup-git-config
 
 ### Python ###
 
-venv_name=".venv"
-
-if [ ! -f "${venv_name}/bin/activate" ]; then
-    prompt_python_venv_creation
-    # This is time consuming but it has to be done
-    source "${SENTRY_ROOT}/scripts/bootstrap-py3-venv"
+if [ ! -x "${HOME}/.local/share/sentry-devenv/bin/devenv" ]; then
+    # Old and busted.
+    venv_name=".venv"
+    if [ ! -f "${venv_name}/bin/activate" ]; then
+        prompt_python_venv_creation
+        # This is time consuming but it has to be done
+        source "${SENTRY_ROOT}/scripts/bootstrap-py3-venv"
+    fi
+else
+    # The new hotness (coming soon).
```

All software is either busted or not ready yet.

```suggestion
    # The new hotness (not ready yet).
```

---

lol